receiver/dockerstatsreceiver/documentation.md

```diff
@@ -678,8 +678,10 @@ Packets sent.
 
 | Name | Description | Values | Enabled |
 | ---- | ----------- | ------ | ------- |
+| container.command | The command executed by the container. | Any Str | false |
```

Comment:
Please add this resource attribute to this table: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/container.md

Reply:
I already created the PR https://github.com/open-telemetry/opentelemetry-specification/pull/3468.